utils/settings.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const homedir = require('os').homedir();
+const path = require('path');
+
+
+// Persistent shell settings
+
+const getShellSettings = () => {
+    const nearPath = path.join(homedir, '.near-config');
+    try {
+        if (!fs.existsSync(nearPath)) {
+            fs.mkdirSync(nearPath);
+        }
+        const shellSettingsPath = path.join(nearPath, 'settings');
```

Comment:
can we have this be `settings.js` please

Reply:
.json would be more appropriate. I don't want to make it .js because it's not actually a javascript file, but I am on board with .json